karma.conf.js

```diff
@@ -6,7 +6,7 @@ var basePath = __dirname;
 module.exports = function (config) {
   const configObject =  {
     basePath: '',
-    frameworks: ['mocha', 'chai', 'sinon-chai'],
+    frameworks: ['mocha', 'chai', 'sinon-chai', 'es6-shim'],
```

Comment:
We have to check this I think now we don't need es6-shim since we decided to output es-6, but that's something I have to double check with @Nasdan if it's on master now.